code/game/machinery/rechargestation.dm

```diff
@@ -183,15 +191,21 @@
 
 
 /obj/structure/machinery/recharge_station/proc/go_out()
-	if(!( src.occupant ))
+	if(!occupant)
 		return
-	//for(var/obj/O in src)
-	// O.forceMove(src.loc)
-	if (src.occupant.client)
-		src.occupant.client.eye = src.occupant.client.mob
-		src.occupant.client.perspective = MOB_PERSPECTIVE
-	src.occupant.forceMove(loc)
-	src.occupant = null
+	var/mob/living/cyborg = occupant
```

Comment:
(Nitpick)
I know that there's other references to 'cyborg'/'borg' in the file, but since the occupant will always be a synthetic here, I feel like it'd be worth changing this variable name from `cyborg` to `synth` instead.